docs/acp/user/app-manifest.md

```diff
@@ -210,6 +220,23 @@ For the manifest, make sure to follow these conditions:
 ```
 </details>
 
+<details>
+<summary>messages example</summary>
+
+```json
+"messages": {
+    "foo": {
+        "type": "text",
+        "data": "bar"
+    },
+    "baz": {
+        "type": "text",
+        "data": "qux"
+    }
+}
+```
+</details>
```

Comment:
Imagine you are a PBC developer and need to provide a manifest for your new app. 
You read this document and see `messages` with some key, type and data. What can you put into the values? is it required? does my application need it?

Can you try to explain on the examples how these `messages` could be used.

Reply:
That's an optional property. In such case you skip it.

Reply:
The question is HOW to use it. Where is the list of available `key`s (onConnect, connectSuccessful or other).
What `type` are available?

Reply:
- Keys should be documented on PBC end, they define what they need.
- The only type supported right now is `text`. That was mentioned in the table.
    > Mandatory. Message type (available types: `text`)

Reply:
@mharacewiat 
Right now only two keys from the FE side are supported

onSave, onDisconnect

They could not be defined by PBC side

Reply:
@matweew What do we do with it?

Reply:
add them to the description of messages and examples

Reply:
They have been added, but I lack clarification - when they trigger and what to expect if we don't define them in the manifest.